src/main/java/com/ss/training/utopia/agent/controller/AgentController.java

```diff
@@ -0,0 +1,50 @@
+package com.ss.training.utopia.agent.controller;
+
+import java.util.List;
+
+import com.ss.training.utopia.agent.entity.Airport;
+import com.ss.training.utopia.agent.service.AgentService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author Trevor Huis in 't Veld
+ */
+@RestController
+@RequestMapping(path="/utopia/agent")
+public class AgentController {
+
+    @Autowired
+	AgentService service;
+
+    @GetMapping(path="/airport/{id}")
+    public ResponseEntity<Airport> getAirport(@PathVariable int id) {
+		Airport airport = null;
+		HttpStatus status = HttpStatus.OK;
+
+		airport = service.readAirport(id);
+		if (airport == null) // no author with the specified ID exists
```

Comment:
Forgot to update the comment.

Also, I don't think the airport will ever be null here. If the airport doesn't exist, the DAO will return an [empty Optional](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html?is-external=true#findById-ID-), which I'm not sure exactly how to handle. If there's a database access problem, it'll presumably throw an exception instead of returning anything.